dll/win32/shell32/CCopyAsPathMenu.cpp

```diff
@@ -167,7 +112,7 @@ CCopyAsPathMenu::QueryContextMenu(HMENU hMenu, UINT indexMenu, UINT idCmdFirst,
           hMenu, indexMenu, idCmdFirst, idCmdLast, uFlags);
 
     if ((uFlags & CMF_NOVERBS) || !(uFlags & CMF_EXTENDEDVERBS))
-        return MAKE_HRESULT(SEVERITY_SUCCESS, 0, idCmdFirst);
+        return MAKE_HRESULT(SEVERITY_SUCCESS, 0, 0);
```

Comment:
Elaborate this for regular persons, please!

Reply:
Unfortunately [the documentation is garbage](https://learn.microsoft.com/en-us/windows/win32/api/shobjidl_core/nf-shobjidl_core-icontextmenu-querycontextmenu#return-value) and what to do this case is unclear. Either and E_ or S_OK. But not a positive number because no items were added so no menu items were used.

Reply:
`IContextMenu::QueryContextMenu`
https://learn.microsoft.com/en-us/windows/win32/api/shobjidl_core/nf-shobjidl_core-icontextmenu-querycontextmenu

Reply:
> If successful, returns an HRESULT value that has its severity value set to SEVERITY_SUCCESS and its code value set to the offset of the largest command identifier that was assigned, plus one. For example, if idCmdFirst is set to 5 and you add three items to the menu with command identifiers of 5, 7, and 8, the return value should be MAKE_HRESULT(SEVERITY_SUCCESS, 0, 8 - 5 + 1). Otherwise, it returns a COM error value.

There are some logical errors.

Reply:
Where code does the difference of IDs affect?

Reply:
CDefaultContextMenu

Reply:
Could you fix it if possible if you can?

If impossible, then more clever guy will fix it in the future.